Readme.md

```diff
@@ -1,1266 +1,32 @@
-For this, the ID of the authorized user and the specific ID of an email which is available in that Gmail account should 
-be provided as parameters. This operation will return a `Message` if successful. Else return an `error`. But, the focus
-of this operation is extracting the attachment and that can be accessed inside the `MessageBodyPart` returned.
-
-```ballerina
-
-// ID of the message to read
-string sentMessageId = "<MESSAGE_ID>";
-
-gmail:Message|error response = gmailClient->readMessage(sentMessageId);
-
-if (response is gmail:Message) {
-   if (response?.msgAttachments is gmail:MessageBodyPart[] ) {
-        log:printInfo("Attachment retrieved " + (<gmail:MessageBodyPart[]>response?.msgAttachments).toStrin());
-   } else {
-        log:printError("No attachment exists for this message");
-   }
-} else {
-    log:printError("Failed to get attachments");
-}
-
-```
-Sample is available at: https://github.com/ballerina-platform/module-ballerinax-googleapis.gmail/blob/master/samples/messages/read_message_with_attachments.bal
-
-### Get attachment
-This sample shows how to read an attachment that is available in the authorized user's account.Similar to above sample, 
-we can to obtain the attachment itself using this operation. It will return only a `MessageBodyPart` as the response if
-successful. Else returns `error`. 
-
-```ballerina
-
-
-
-// ID of the message where attachment belongs to.
-string sentMessageId = "<MESSAGE_ID>"; 
-
-// ID of the attachment
-string readAttachmentFileId = "<ATTACHMENT_ID>";
-
-gmail:MessageBodyPart|error response = gmailClient->getAttachment(sentMessageId, readAttachmentFileId);
-
-if (response is gmail:MessageBodyPart) {
-    log:printInfo("Attachment " + response.toString());
-} else {
-    log:printError("Failed to get the attachments : "+ response.message());
-}
-
-```
-Sample is available at: https://github.com/ballerina-platform/module-ballerinax-googleapis.gmail/blob/master/samples/messages/get_attachment.bal
-
-### Trash or un-trash message
-This sample shows how to trash or un-trash an email that is available in the authorized user's account. When you trash a 
-message, it stays in your Trash folder for 30 days. After that time, it will be permanently deleted. So if you want to 
-recover the email you have deleted within these 30 days, you can do so using the un-trash capability of the connector. 
-These two operations returns `Message` record if the operations are successful. Else returns `error`.
-
-```ballerina 
-// Moves the specified message to the trash.
-
-
-// ID of the message to trash or un-trash.
-string sentMessageId = "<MESSAGE_ID>"; 
+enables users to send and receive emails over the Internet.
 
-gmail:Message|error trash = gmailClient->trashMessage(sentMessageId);
+Gmail connector connects to Gmail API from [Ballerina](https://ballerina.io/). This connector provides the capability to programatically handle Gmail related operations and listen changes in the mailbox.
```

Comment:
from - > via.
listen-> watch for changes in the Gmail mailbox.

Reply:
Fixed by ddd90073069938a3660747b110c4643c53e57b4f